static/views/property-meta.php

```diff
@@ -11,7 +11,12 @@
     $meta_value = do_shortcode(html_entity_decode(get_post_meta( $post_id, $meta_slug, true )));
     if(trim($meta_value) != ""):
```

Comment:
@raj-uimatic also minor, but once again, can play it safer here. If if $meta_value comes back an integer or something? Always try do be safe with `isset()`, `is_string()`, etc.